PowerShell Contributing.md

```diff
@@ -0,0 +1,479 @@
+
+### 6. Contribute Help content
+
+> Requires
+> 
+> - GitHub account
+> - Knowledge of Markdown
+> 
+> Optional
+> 
+> - Git and some knowledge of it
+> - Knowledge of GitHub workflow (fork > clone > PR)
+
+You can contribute by writing, improving, or expanding help files for Get-Help.
+In particular, more examples for cmdlets are always welcome.
+
+Head over to the <https://github.com/MicrosoftDocs/Powershell-Docs> repo, and read their [contributing manual](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) to get started.
+
+> Tip: PowerShell Help is maintained for PowerShell versions 5.1 and higher. You may be asked to update help files for more than one version. 
+
+To do small contributions like correcting typos, just navigate to that file, click the 'Edit' button, and GitHub will create a copy (fork) on your own GitHub for you.
+When you are satisfied with your changes, you can save them, and submit a pull request to the _staging_ branch of the PowerShell-Docs repository.
+
+For bigger contributions, you are better off cloning your fork locally to your computer.
+This enables you to use VSCode and other editing tools.
+A guide on how to do that can be found [here](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/1-GET-STARTED.md#making-major-edits-to-existing-topics).
+
+With more examples, less typos, clearer wording, PowerShell can be easier to learn and use for everyone!
+
+### 7. Write Pester tests
+
+> Requires
+> 
+> - GitHub account
+> - Sign [Microsoft Contributor License Agreement](https://cla.opensource.microsoft.com/)
+> - Git and some knowledge of it
+> - Knowledge of GitHub workflow (fork > clone > PR)
+> - Know how to run tests
+> - Pester 4.8 or higher 
+> - Pester knowledge 
+
+Within the PowerShell project, many parts are tested automatically.
+A big part of that is done with Pester.
+See [here](https://github.com/PowerShell/PowerShell/tree/master/test/powershell).
+
+> Tip: At the time of writing, Pester version 4.8.0 is used in the PowerShell project [`build.psm1` module](https://github.com/PowerShell/PowerShell/blob/a335063f1cb6596636031c2dc064fdf29924c5a7/build.psm1#L666).
+This version pin will change in the future, as new versions of Pester become available.  
+
+Many Cmdlets could use better or more extensive tests to verify they keep working correctly if someone changes them in the future.
+So the Pester tests for Cmdlets are kind of a safeguard against regression.
+When there are more/better tests for Cmdlets, you can possibly prevent future bugs!
+
+See also [Details about contributing to the PowerShell/PowerShell repo](#Details-about-contributing-to-the-PowerShellPowerShell-repo), especially the section on how to run tests.
+
+### 8. Write C# code for PowerShell
```

Comment:
Can you add a bit of explanation of what the link between C# and PS is? There's a lot of how here and I could use a little why..

Reply:
The short version is: PowerShell as an engine and a language is entirely written in C#. 🙂

Reply:
Great point. I'll add that as the first sentence to this chapter.